<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1468</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>In-Cabin Camera Monitor &amp; Data Extractor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="window_camera_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="window_camera_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Camera #1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>0</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>System Status</string>
    </property>
    <widget class="QProgressBar" name="progress_gpu_usage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU-0 Usage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU-0 Mem. Usage</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progress_gpu_mem_usage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1468</width>
     <height>43</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionStart_Recording"/>
    <addaction name="actionStop_Recording"/>
    <addaction name="separator"/>
    <addaction name="actionCapture_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHPE">
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QMenu" name="menuHPE_Model">
     <property name="title">
      <string>HPE Model</string>
     </property>
     <addaction name="actionYOLOv8n_pose"/>
     <addaction name="actionYOLOv8s_pose"/>
     <addaction name="actionYOLOv8m_pose"/>
     <addaction name="actionYOLOv8l_pose"/>
     <addaction name="actionYOLOv8x_pose"/>
    </widget>
    <addaction name="menuHPE_Model"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Keypoints"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionConnect_All"/>
   </widget>
   <addaction name="menuControl"/>
   <addaction name="menuHPE"/>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart_Recording">
   <property name="text">
    <string>Start Video Recording</string>
   </property>
  </action>
  <action name="actionStop_Recording">
   <property name="text">
    <string>Stop Video Recording</string>
   </property>
  </action>
  <action name="actionCapture_Image">
   <property name="text">
    <string>Capture Image(*.png)</string>
   </property>
  </action>
  <action name="actionShow_Keypoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Keypoints</string>
   </property>
  </action>
  <action name="actionYOLOv8n_pose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>YOLOv8n-pose</string>
   </property>
  </action>
  <action name="actionYOLOv8s_pose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>YOLOv8s-pose</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Meta+X</string>
   </property>
  </action>
  <action name="actionYOLOv8m_pose">
   <property name="text">
    <string>YOLOv8m-pose</string>
   </property>
  </action>
  <action name="actionYOLOv8l_pose">
   <property name="text">
    <string>YOLOv8l-pose</string>
   </property>
  </action>
  <action name="actionYOLOv8x_pose">
   <property name="text">
    <string>YOLOv8x-pose</string>
   </property>
  </action>
  <action name="actionConnect_All">
   <property name="text">
    <string>Connect All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
